editor/plugins/bone_map_editor_plugin.cpp

```diff
@@ -35,6 +35,7 @@
 #include "editor/import/post_import_plugin_skeleton_rest_fixer.h"
 #include "editor/import/post_import_plugin_skeleton_track_organizer.h"
 #include "editor/import/scene_import_settings.h"
+#include "modules/regex/regex.h"
```

Comment:
Like all modules, `regex` can be disabled so you'd need to guard against that:
```
#include "modules/modules_enabled.gen.h" // For regex.
#ifdef MODULE_REGEX_ENABLED

#include "modules/regex/regex.h"
```

And then I guess you'd have to bypass the auto-mapping logic when `#ifndef MODULE_REGEX_ENABLED`.

Reply:
Fixed.